From e24dddb9faedc7065e8011c4e043f3991cc0ef13 Mon Sep 17 00:00:00 2001
From: "Le.Ma" <Le.Ma@amd.com>
Date: Tue, 24 Oct 2017 10:30:28 +0800
Subject: [PATCH 3917/5855] drm/amd/powerplay: Rename WM_COUNT to avoid
 redeclaration for dkms build

Another declaration is defined in dm_pp_smu.h.

Change-Id: I4bd9584a2121cdd5ff5547caddc646b07e27af31
Signed-off-by: Le.Ma <Le.Ma@amd.com>
Reviewed-by: Xiaojie Yuan <Xiaojie.Yuan@amd.com>
---
 drivers/gpu/drm/amd/powerplay/inc/smu10_driver_if.h | 4 ++--
 drivers/gpu/drm/amd/powerplay/inc/smu9_driver_if.h  | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/gpu/drm/amd/powerplay/inc/smu10_driver_if.h b/drivers/gpu/drm/amd/powerplay/inc/smu10_driver_if.h
index dea8fe9..1c04cf6 100644
--- a/drivers/gpu/drm/amd/powerplay/inc/smu10_driver_if.h
+++ b/drivers/gpu/drm/amd/powerplay/inc/smu10_driver_if.h
@@ -62,11 +62,11 @@ typedef struct {
 typedef enum {
 	WM_SOCCLK = 0,
 	WM_DCFCLK,
-	WM_COUNT,
+	WM_COUNT_PP,
 } WM_CLOCK_e;
 
 typedef struct {
-	WatermarkRowGeneric_t WatermarkRow[WM_COUNT][NUM_WM_RANGES];
+	WatermarkRowGeneric_t WatermarkRow[WM_COUNT_PP][NUM_WM_RANGES];
 	uint32_t              MmHubPadding[7];
 } Watermarks_t;
 
diff --git a/drivers/gpu/drm/amd/powerplay/inc/smu9_driver_if.h b/drivers/gpu/drm/amd/powerplay/inc/smu9_driver_if.h
index 2818c98..0255184 100644
--- a/drivers/gpu/drm/amd/powerplay/inc/smu9_driver_if.h
+++ b/drivers/gpu/drm/amd/powerplay/inc/smu9_driver_if.h
@@ -339,12 +339,12 @@ typedef struct {
 typedef enum {
   WM_SOCCLK = 0,
   WM_DCEFCLK,
-  WM_COUNT,
+  WM_COUNT_PP,
 } WM_CLOCK_e;
 
 typedef struct {
   /* Watermarks */
-  WatermarkRowGeneric_t WatermarkRow[WM_COUNT][NUM_WM_RANGES];
+  WatermarkRowGeneric_t WatermarkRow[WM_COUNT_PP][NUM_WM_RANGES];
 
   uint32_t     MmHubPadding[7]; /* SMU internal use */
 } Watermarks_t;
-- 
2.7.4

